apps/registry/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM registry:2.8.1
+
+ADD https://raw.githubusercontent.com/sunodo/sunodo/main/apps/api/keys/registry.crt /keys/registry.crt
```

Comment:
Do you want to add this to the global Dockerfile at root, as a `registry` target?
Then the context is at root and you can copy the keys from the project.

I know this is probably temporary

Reply:
I prefer smaller self-container `Dockerfiles`, but I can do that if you think we should go the "single Dockerfile with a lot of targets" path.

Reply:
I just don't like adding the file from GitHub as you did